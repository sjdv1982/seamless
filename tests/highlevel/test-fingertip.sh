#!/bin/bash

echo 'Part 1'
python3 test-fingertip1.py
echo ''

echo 'Part 2'
python3 test-fingertip2.py
echo ''

echo 'Part 3'
python3 test-fingertip1.py
echo ''

export SEAMLESS_READ_BUFFER_SERVERS=""
export SEAMLESS_WRITE_BUFFER_SERVER=""
echo 'Part 4'
python3 test-fingertip1.py
echo ''

echo 'Part 5'
python3 test-fingertip2.py
echo ''