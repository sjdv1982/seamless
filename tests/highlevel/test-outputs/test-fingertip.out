Part 1
Waiting for: Seamless transformer: .func.tf
1ec587570668eb78b2449329ffd174fc38472f149811d9206814b741f0d7fc40
*************************************************
* Result
*************************************************
<checksum 1ec587570668eb78b2449329ffd174fc38472f149811d9206814b741f0d7fc40>
*************************************************
Execution time: 0.0 seconds
*************************************************
<Silk: 860 >
5.2 seconds elapsed

Part 2
860
0.0 seconds elapsed

Part 3
1ec587570668eb78b2449329ffd174fc38472f149811d9206814b741f0d7fc40
None
<Silk: 860 >
0.1 seconds elapsed

Part 4
1ec587570668eb78b2449329ffd174fc38472f149811d9206814b741f0d7fc40
None
<Silk: 860 >
5.3 seconds elapsed

Part 5
860
5.0 seconds elapsed

